2018-08-21 23:24:07,251 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-21 23:26:30,149 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-21 23:30:37,325 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-21 23:31:14,473 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-21 23:32:14,322 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-21 23:41:39,759 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-21 23:49:11,371 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-21 23:58:16,158 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-21 23:59:58,152 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 00:05:40,421 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 00:19:40,326 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 00:20:37,566 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 00:24:03,117 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 00:25:51,784 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 00:27:45,437 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 00:29:08,916 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 00:31:09,368 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 00:31:42,588 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 00:45:24,205 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 00:46:44,548 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 00:48:49,918 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 00:55:18,962 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 00:56:27,667 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 00:58:25,643 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 01:01:13,592 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 01:03:15,106 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 01:03:47,117 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 01:07:13,938 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 01:08:37,643 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 01:09:36,220 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 01:14:41,614 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 01:14:50,490 ERROR: Exception on /landing [GET] [in /usr/local/lib/python2.7/dist-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python2.7/dist-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/root/microblog_tryout/app/routes.py", line 24, in landing
    return render_template("landing.html", status=status, title='Landing Page', posts=posts)
  File "/usr/local/lib/python2.7/dist-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python2.7/dist-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/root/microblog_tryout/app/templates/landing.html", line 11, in template
    <button name="Getting Started" type="submit">Resume {% status %}</button>
TemplateSyntaxError: Encountered unknown tag 'status'. Jinja was looking for the following tags: 'endif'. The innermost block that needs to be closed is 'if'.
2018-08-22 01:14:52,031 ERROR: Exception on /landing [GET] [in /usr/local/lib/python2.7/dist-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python2.7/dist-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/root/microblog_tryout/app/routes.py", line 24, in landing
    return render_template("landing.html", status=status, title='Landing Page', posts=posts)
  File "/usr/local/lib/python2.7/dist-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python2.7/dist-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/root/microblog_tryout/app/templates/landing.html", line 11, in template
    <button name="Getting Started" type="submit">Resume {% status %}</button>
TemplateSyntaxError: Encountered unknown tag 'status'. Jinja was looking for the following tags: 'endif'. The innermost block that needs to be closed is 'if'.
2018-08-22 01:15:54,055 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 01:33:56,280 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 01:34:08,786 ERROR: Exception on /startlearning [GET] [in /usr/local/lib/python2.7/dist-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python2.7/dist-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/root/microblog_tryout/app/routes.py", line 104, in sl
    return redirect(url_for('learn'))
  File "/usr/local/lib/python2.7/dist-packages/flask/helpers.py", line 356, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 2061, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/helpers.py", line 345, in url_for
    force_external=external)
  File "/usr/local/lib/python2.7/dist-packages/werkzeug/routing.py", line 1776, in build
    raise BuildError(endpoint, values, method, self)
BuildError: Could not build url for endpoint 'learn'. Did you mean 'learning' instead?
2018-08-22 01:36:12,617 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
2018-08-22 01:37:33,782 INFO: Microblog startup [in /root/microblog_tryout/app/__init__.py:45]
